packages/ses/docs/lockdown.md

```diff
@@ -617,6 +709,15 @@ lockdown({ overrideTaming: 'min' }); // Minimal mitigations for purely modern co
 lockdown({ overrideTaming: 'severe' }); // More severe legacy compat
 ```
 
+If `lockdown` receives no election for `stackFiltering`, it will
```

Comment:
```suggestion
If `lockdown` receives no election for `overrideTaming`, it will
```
Other similar cut/paste errors in this same position below. Please check them all.